_posts/2016-04-27-discounted-goruco-tickets-for-underrepresented-people.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+excerpt_separator: <!--more-->
+---
+
+<p>
+  This year we are offering discounted tickets for those who self-identify
+  as underrepresented in technology via a discount code.</p>
```

Comment:
As I was reading it, I think "self-identify" is redundant. Also I'm not sure we need to mention the discount code here. Suggested edit:

_This year we are offering discounted tickets for those who identify as belonging to a group that is underrepresented in the technology industry._